docs/reference/changelog.rst

```diff
@@ -4,14 +4,19 @@
 Changelog
 *********
 
-5.2.1 (2025-XX-YY)
+5.2.1 (2025-05-07)
```

Comment:
FYI this will probably overlap with #754 and be thus be part of 5.3.0.

Reply:
Will reconcile accordingly.